test/unit/ui/marker.test.js

```diff
@@ -465,3 +465,69 @@ test('Marker with draggable:true does not error if removed on mousedown', (t) =>
     t.ok(map.fire('mouseup'));
     t.end();
 });
+
+test('Marker can set rotationAlignment and pitchAlignment', (t) => {
```

Comment:
We should probably add a test that ensures setting `rotationAlignment === 'map'` and `pitchAlignment === 'auto'` results in `pitchAlignment === 'map'`. Both through the options object and by calling `set{Rotation/Pitch}Alignment`

Reply:
Added a couple tests. One is for the option on initialization, the other is for the getter/setters.